From 7391cef72e2076dc5807367451311c04323138fe Mon Sep 17 00:00:00 2001
From: Thierry Reding <treding@nvidia.com>
Date: Wed, 2 Apr 2014 09:47:01 +0200
Subject: [PATCH 262/430] of: Use NULL for pointers

Commit 4485681939b9 (of/fdt: Clean up casting in unflattening path)
modified unflatten_dt_node() to take a void * for the mem parameter
instead of an unsigned long. One of the call sites wasn't updated.

Signed-off-by: Thierry Reding <treding@nvidia.com>
Signed-off-by: Grant Likely <grant.likely@linaro.org>

Conflicts:
	drivers/of/fdt.c
(cherry picked from commit d2d3d7cd81e90e1ffac1a6eed7b3edcbf11f4c97)
Signed-off-by: Zumeng Chen <zumeng.chen@windriver.com>
---
 drivers/of/fdt.c |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/drivers/of/fdt.c b/drivers/of/fdt.c
index 1fbeab2..ac48c70 100644
--- a/drivers/of/fdt.c
+++ b/drivers/of/fdt.c
@@ -325,7 +325,7 @@ static void __unflatten_device_tree(void *blob,
 
 	/* First pass, scan for size */
 	start = 0;
-	size = (unsigned long)unflatten_dt_node(blob, 0, &start, NULL, NULL, 0);
+	size = (unsigned long)unflatten_dt_node(blob, NULL, &start, NULL, NULL, 0);
 	size = ALIGN(size, 4);
 
 	pr_debug("  size is %lx, allocating...\n", size);
-- 
1.7.5.4

